content/partials/build-versioning-ios.md

```diff
@@ -0,0 +1,30 @@
+---
+---
+
+In addition to the App Store credentials configured earlier, in order to get the latest build number from App Store or TestFlight, you will also need the **Application Apple ID**. This is an automatically generated ID assigned to your app and it can be found under **General > App Information > Apple ID** under your application in App Store Connect.
```

Comment:
Partials should be standalone so they can be reused anywhere. Therefore, I would re-word this slightly to remove the reference to things that were configured earlier.